apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.ts

```diff
@@ -75,8 +75,7 @@ export function AssessmentQuestion({
         )}
       </head>
       <body>
-        ${renderEjs(import.meta.url, "<%- include('../../partials/navbar'); %>", resLocals)}
-        ${GradingConflictModal()}
+        ${Navbar({ resLocals })} ${GradingConflictModal()}
```

Comment:
To preserve line breaks we've sometimes done this:
```suggestion
        ${[
          Navbar({ resLocals }),
          GradingConflictModal(),
        ]}
```

Reply:
👍 I'm actually a big fan of this!

Reply:
I've seen you do this, I was wondering why...

Reply:
I've done it in this page and tested it, but to be able to avoid pushing this PR further down the line I haven't changed other pages. I think we can change those progressively in future PRs.

Reply:
Unfortunately this didn't survive lint, since the array itself fits in a single line... I've reverted it and I'll merge without it.

Reply:
For reference: if splitting lines is actually desirable, this [could be converted to](https://prettier.io/blog/2018/11/07/1.15.0#whitespace-sensitive-formatting):

```html
        <!-- display: block -->
        ${Navbar({ resLocals })}
        <!-- display: block -->
        ${GradingConflictModal()}
```

I won't do this here, though.